pegasus/sites.v3/advocacy.code.org/public/index.haml

```diff
@@ -64,6 +64,13 @@ video_player: false
 
     .coalition
       %h2 Our Coalition
+      %br
```

Comment:
Surprised that we need a blank line here since the `h2` should have a bunch of margin below it already.

Reply:
I thought the spacing was too tight compared to the other button. Here's a side by side comparison.

Without %br | With %br
-------------|---------
![image](https://user-images.githubusercontent.com/1920530/38756907-ed4d3d62-3f1f-11e8-8765-265cf9a5ce53.png)|![image](https://user-images.githubusercontent.com/1920530/38756888-e0ce03e6-3f1f-11e8-9c0e-3d9409d3afad.png)